benchmarks/asv.conf.json

```diff
@@ -31,7 +31,6 @@
         "Cython": [],
         "numpy": [],
         "scipy": [],
-	    "six": [],
```

Comment:
You can’t remove six from asv because it is needed for testing the older versions.

Reply:
More precisely, we need to check what should be there and what we are actually using. I think i only have NumPy and cython in the actual cfg file, see the benchmark repo/wiki.  However, just leave six here until this gets sorted out in a separate asv cleanup PR. Better to leave the history in place because ASV needs to build historical versions of the code.

Reply:
Makes sense, done in e5a3fe1